.github/workflows/test.yml

```diff
@@ -108,6 +108,40 @@ jobs:
           ## If no one connects after 5 minutes, shut down server.
           wait-timeout-minutes: 5
 
+  devel-tests:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v5
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: '3.12'
+      - name: Setup libvirt for Vagrant
+        uses: voxpupuli/setup-vagrant@v0
+      - name: Install Ansible
+        run: pip install --upgrade ansible-core
+      - name: Setup environment
+        run: ./setup-environment
+      - name: Start VMs
+        run: |
+          ./forge vms start
+      - name: Configure repositories
+        run: |
+          ./forge setup-repositories
+      - name: Run deployment
+        run: |
+          ./forge deploy-dev --foreman-development-enabled-plugin foreman_ansible --foreman-development-enabled-plugin katello
```

Comment:
`katello` is part of the `default` definition, so you don't need to repeat it here

Reply:
That's how it should work, but not what I was encountering. Aaand it's because we do not actually support default values at the argparse level:

https://github.com/theforeman/obsah/blob/master/obsah/__init__.py#L383

I am assuming this is because we wanted to forge this information into the Ansible layer.

That leaves me with two options I can think of:

1. Change this behaviour in Obsah
2. Change to having variable inside the role for defaults and separate one for external CLI setting and then append the two together

Reply:
Right, why did I think we can set `default` in obsah? :thinking: